examples/js/controls/FirstPersonControls.js

```diff
@@ -30,6 +30,7 @@ THREE.FirstPersonControls = function ( object, domElement ) {
 	this.verticalMax = Math.PI;
 
 	this.autoSpeedFactor = 0.0;
+	this.forward = new THREE.Vector3( 0, 0, - 1 );
```

Comment:
What happens if the user changes this?

Reply:
Then the forward vector is differently. This is necessary if the user passes a ordinary `Object3D` to the controls.

Reply:
if used to control other object, it becomes ThirdPersonControls :)

Reply:
> I believe FirstPersonControls is designed to control a camera

Why put a limit on what it can control, if it's as simple as exposing `this.forward`?

Reply:
If we do decide to impose this limit then we should change this line:

``` js 
THREE.FirstPersonControls = function ( object, domElement ) 
```
to
``` js
THREE.FirstPersonControls = function ( camera, domElement ) 
```

And probably also for other controls too, because `object` suggests that any `Object3D` can be used.

Reply:
@looeee Absolutely!

Reply:
@looeee @Mugen87 I think the proper question is why the hell forward needs to be different for camera vs other objects? Why can't it just be 0,0,-1 for everything?

Reply:
That's more or less a WebGL standard. (0,0,1) for ordinary objects and (0,0,-1) for cameras, see https://stackoverflow.com/questions/33407209/why-does-the-camera-face-the-negative-end-of-the-z-axis-by-default?rq=1

Reply:
@looeee I have hardwired the forward vector for now since the controls do not fully support arbitrary objects yet, see https://jsfiddle.net/q50o1r97/1/. If there is any interest in changing this, I suggest to do this with a separate PR. Same for renaming `object` to `camera` in control classes.

Reply:
@Mugen87 OK, but I think it would be good to have clarity on this. Either we do support controlling arbitrary objects with controls, or we don't. 

If we do, then it should be noted somewhere - even just as a comment in the files. 

If we don't, then we can just rename `object` to `camera`. 
I'm OK with either.